backend/config/packages/dev/monolog.yaml

```diff
@@ -9,7 +9,7 @@ monolog:
     handlers:
         main:
             type: stream
-            path: '%kernel.logs_dir%/%kernel.environment%.log'
+            path: php://stderr
```

Comment:
Check this file: https://github.com/Runroom/archetype-symfony/blob/master/config/packages/monolog.yaml#L26-L27

Reply:
With json you get a better output for local (on prod you should not change it, because you might want to  use a log aggregator and it prefers a default format)
And you don't need doctrine events, since you look at that using the profiler.

Reply:
thank you @jordisala1991 !